### ISSUE TITLE
torque qstat has libtk error

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#554]
**Reported by naughtont on 6 Jul 2009 21:49 UTC**
The 'qstat' command included with the torque-oscar-client-2.1.10-3' (x86) has a shared library problem (see msg below).  

Testing environment is Ubuntu 8.04.3 LTS (x86)  
NOTE: The rest of Torque is not (currently) auto-configured by OSCAR.

```

 tjn@node0:$ qstat --help
 qstat: error while loading shared libraries: libtk8.4.so.0: cannot
open shared object file: No such file or directory

```

**Comment by dikim33** [comment]
**Comment by naughtont on 22 Jul 2009 18:26 UTC**
Update: This appears to work if you manually install tk8.4.  So the binary pkgs don't appear to have the proper requires to pull in the library, but they are linked against the .so.

Note, I tested with following...
```
 tjn@node0:$ qstat --help
 qstat: error while loading shared libraries: libtk8.4.so.0: cannot
open shared object file: No such file or directory

 tjn@node0:$ sudo apt-get install tk8.4 tk8.4-dev
    ...lots of output here

 tjn@node0:$ qstat --help
 usage: 
 qstat [[-1](-f]) [site_specific](-W) [ job_identifier... | destination... ]
 qstat [[-u user](-a|-i|-r|-e]) [[-1](-n)] [[-G|-M](-s]) [[job_id... | destination...](-R])
 qstat -Q [[-1](-f)] [site_specific](-W) [ destination... ]
 qstat -q [[ destination... ](-G|-M])
 qstat -B [[-1](-f)] [site_specific](-W) [ server_name... ]
```

**Comment by dikim33** [comment]
**Comment by valleegr on 5 Aug 2009 20:53 UTC**
(In [8715]) This check-in fixes #555 - Fix the missing dependency to Tk for the GUI 
package for Torque.
